.github/CODEOWNERS

```diff
@@ -1,5 +1,17 @@
+# Team Review
 * @robambalu @AdamGlustein @svatasoiu @alexddobkin
 
-.github @robambalu @timkpaine @ngoldbaum @alexddobkin @AdamGlustein @svatasoiu
-.github/CODEOWNERS @robambalu
+# Packaging code
+CMakeLists.txt @robambalu @AdamGlustein @svatasoiu @alexddobkin @timkpaine
```

Comment:
we should have at least one quansight rep on these lists

Reply:
@czgdp1807 you cool to be in this list?

Reply:
Yes. Sure. I am fine with me being in the list. Thanks @timkpaine